console-subscriber/src/builder.rs

```diff
@@ -56,6 +56,10 @@ pub struct Builder {
     /// Any scheduled times exceeding this duration will be clamped to this
     /// value. Higher values will result in more memory usage.
     pub(super) scheduled_duration_max: Duration,
+
+    #[cfg(feature = "grpc-web")]
+    /// Whether to enable the grpc-web support.
```

Comment:
Typically the `#[cfg]` goes below the doc comment:
```suggestion
    /// Whether to enable the grpc-web support.
    #[cfg(feature = "grpc-web")]
```